tests/framework/installer/ceph_installer.go

```diff
@@ -680,6 +680,13 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(manifests ...CephManifests)
 			continue
 		}
 
+		// remove mon finalizers
+		logger.Infof("example")
+		cluster.RemoveMonSecretFinalizer(h.k8shelper.MakeContext().Client, namespace)
+		cluster.RemoveMonConfigMapFinalizer(h.k8shelper.MakeContext().Client, namespace)
```

Comment:
I am getting err: `suite.go:63: test panicked: runtime error: invalid memory address or nil pointer dereference`, from this call.

Reply:
The `MakeContext()` method does not initialize the `Client`, so it would cause the nil pointer inside the `RemoveMonConfigMapFinalizer()` method

Reply:
thanks, got it